functions/contrib/go/annotate-apply-time-mutations/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/GoogleContainerTools/kpt-functions-catalog/functions/contrib/go/annotate-apply-time-mutations
+
+go 1.16
+
+require (
```

Comment:
Please ensure you have run `go mod tidy` to prune unnecessary dependencies.

Reply:
done